pages/oss.md

```diff
@@ -13,7 +13,7 @@ As previously mentioned, a number of private sector companies have shifted some
 
 * The General Services Administration's (GSA) 18F<sup id="fnr28"><a href="#fn28">28</a></sup> and the Consumer Financial Protection Bureau (CFPB)<sup id="fnr29"><a href="#fn29">29</a></sup> – Both of these government organizations have policies that establish a default position to publicly publish all custom code developed by or for the organization—whether developed in-house by Federal staff or through negotiated agreements—with limited exceptions;
 
-* The Department of Defense (DoD) – This government agency issued guidance<sup id="fnr30"><a href="#fn30">30</a></sup> dating back to 2009 that, among other things, clarifies the use of OSS at DoD and makes clear that OSS products are on equal footing with their proprietary counterparts in terms of procurement and usage; and
+* The Department of Defense (DoD) – This government agency issued guidance<sup id="fnr30"><a href="#fn30">30</a></sup> dating back to 2009 that, among other things, clarifies the use of OSS at DoD and makes clear that OSS products are on equal footing with their proprietary counterparts in terms of procurement, usage, and security; and
```

Comment:
From [the DoD memo](http://dodcio.defense.gov/Portals/0/Documents/OSSFAQ/2009OSS.pdf#page=4), paragraph 2(b)(1)(i) of Attachment 2:

> The continuous and broad peer-review enabled by publicly available source code supports software reliability and security efforts through the identification and elimination of defects that might otherwise go unrecognized by a more limited core development team.